elements/the-graph-nav.html

```diff
@@ -94,52 +94,14 @@
           this.set('style.cursor', 'move');
           this.set('style.width', this.width + 'px');
           this.set('style.height', this.height + 'px');
-          // HACK way to make PolymerGestures work for now
-          if (window.PolymerGestures) {
-            var noop = function () {
-            };
-            window.PolymerGestures.addEventListener(this, 'track', noop);
-            window.PolymerGestures.addEventListener(this, 'trackend', noop);
-            window.PolymerGestures.addEventListener(this, 'tap', noop);
-          }
-          // Pan graph by dragging map
```

Comment:
@bergie is this feature removed or what?

Reply:
It seems this never worked after we went Polymer 1.x

Reply:
Might not have anything to do with Polymer 1.x, but with the-graph. There is a plain-React way to do it alo, see the-graph full example.